### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by amn41**
### Brief description. What is this change?

Uses a more standard Rasa NLU pipeline that includes a pretrained model. 
Not using DIET or anything with a lot of hyperparameters that could be tweaked, since this is mean to be a baseline.

Has a meaningful impact on results, e.g. accuracy on banking77 jumps from `0.671` to `0.797`.

### Setup information

Require an extra dependency to be installed: transformers.

### CODE REVIEW

**Comment by Diophontine** [comment]
https://github.com/voiceflow/vfnlu-tests/pull/8 Updated in this PR
